src/GrumPHP/Task/Behat.php

```diff
@@ -49,31 +50,19 @@ public function canRunInContext(ContextInterface $context)
     public function run(ContextInterface $context)
     {
         $files = $context->getFiles()->name('*.php');
+        // We don't care about changed files here, we want to run the entire suit every time
```

Comment:
This comment is probably a leftover from something else? It seems that we don't run if no files are changed, looking at the next few lines.